### ISSUE TITLE
using a class from another file in the initial context causes SyntaxError: Unexpected token

### ISSUE THREAD

**Issue by redappleorigin** [mattpocock/xstate-codegen#16]
I have an example branch here: https://github.com/redappleorigin/xstate-codegen/tree/imported-class-syntax-error

It is forked off of xstate-codegen@0.1.0-next.7

I added a new folder and file to examples 'models/example.model.ts' and modified fetchMachine.machine.ts so that it specifies an initialContext with one of the properties being `new ExampleModel()`. This causes a `SyntaxError: Unexpected token 'export'`

**Comment by redappleorigin** [comment]
The interface referencing the imported class is fine and does not throw an error. It seems that trying to instantiate my ExampleModel class anywhere in the machine file will cause that SyntaxError.

**Comment by redappleorigin** [comment]
I'm using node v14.2.0 on a windows 10 machine.

**Comment by mattpocock** [comment]
@redappleorigin I'm off on holiday today, so this fix might have to wait a few days. @Andarist would you know a reason why importing a class might mess with the extractMachines rollup config?

**Comment by mattpocock** [comment]
@redappleorigin Do you have a local babel config in the dir where you're running the codegen tool?

**Comment by redappleorigin** [comment]
@mattpocock no sir

**Comment by Andarist** [comment]
Not sure why this wouldn't be working - Rollup should handle your model file and transpile it down to CJS. I'm rewriting the extraction from the current Babel+Rollup+execute setup to a pure TS-based one, so I don't think it's worth looking into it now as it should just get solved soon-ish anyway.

**Comment by redappleorigin** [comment]
Did the example I provided have the same issue for you?

**Comment by redappleorigin** [comment]
Just curious if maybe it is my machine somehow.

**Comment by rjdestigter** [comment]
I'm running into this issue as well

**Comment by redappleorigin** [comment]
@Andarist I don't suppose you have a bleeding edge branch I could work off, would you?

**Comment by mattpocock** [comment]
@redappleorigin back from hols, so will have a look at your example tomorrow. Agree with @Andarist that this is odd.

**Comment by rjdestigter** [comment]
In my case it has to do with an importet enum. As soon as I import it from another file rather than declare it in the same module where it is being used the `SyntaxError: Unexpected token 'export'` error shows up.

**Comment by rjdestigter** [comment]
It's because the configuration for `external` that is passed to _rollup_ isn't working correctly. It's marking files that start with a forward slash (`/`) as external when they're not.

**Comment by mattpocock** [comment]
@rjdestigter would this work?

```js
// Tests for '.' or '/' at the start of a string
const internalRegex = /^(\.|\/)/;
external: (id) => !internalRegex.test(id),
```

**Comment by redappleorigin** [comment]
@mattpocock in my case I am on windows, so a slight change was required:

```ts
// Tests for '.', '/', or some drive letter followed immediately by a ':' at the start of a string
const internalRegex = /^(\.|\/|\w:)/.test(id);
external: (id) => !internalRegex.test(id),

**Comment by mattpocock** [comment]
Released under 0.1.0-next.8
